README.md

```diff
@@ -2,40 +2,33 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-Populate your `.env.local` with `cp .env.example .env.local`
+Copy `.env.example` to `.env` and populate it with missing/relevant environment variables.
 
 ### Docker
 
 ```sh
 docker compose up
 ```
 
-Note: installing new packages require image re-build with `docker compose up --build`
-
-### Local
-
-1. Start `postgres` database somewhere and populate your `.env` with relevant environment variables.
-2. Install dependencies with `npm install`
-3. Run `npm run dev`
-
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `app/page.js`. The page auto-updates as you edit the file.
+Note: installing new packages require image re-build with `docker compose up --build`
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+### Local
 
-#### Linting & Formatting
+#### Requirements
 
-Linting is enabled on pre-commit hook, and manually you can run it with `npm run lint` or
-`npm run lint:fix` to automatically apply fixes.
+- Node.js 20+
+- Postgres 16+ running somewhere, and `.env` database variables populated
 
-## Git conventions & guides
+1. `npm install`
+2. `npm run dev`
```

Comment:
I guess we need one more step between these two for running migrations.

Reply:
added